Information/CenterOfMass.FCMacro

```diff
@@ -363,30 +377,51 @@ class CenterofmassWidget(QtGui.QWidget):
             self.resultCdG[axis].setObjectName('center of mass')
             self.resultCdG[axis].setReadOnly(True)
 
+        cdgLayout = QtGui.QHBoxLayout()
+        cdgLayout.addWidget(label_CdG)
+        for axis in range(3):
+            cdgLayout.addWidget(self.resultCdG[axis])
+        cdgGroupBox = QtGui.QGroupBox()
+        cdgGroupBox.setLayout(cdgLayout)
+        layout.addWidget(cdgGroupBox)
+
+	# viewGroupBox
         showCoM = QtGui.QCheckBox(toolTip='Show center of mass')
         if self.doc.getObject('CenterOfMass'):
             showCoM.setChecked(True)
         showCoM.setIcon(QtGui.QIcon(':/icons/Std_ToggleVisibility.svg'))
         showCoM.setIconSize(g_icon_size)
         showCoM.stateChanged.connect(self.on_stateChanged_showCoM)
 
-        self.changeRadius = QtGui.QSlider(QtGui.Qt.Vertical)
+        self.switchVolumMass = QtGui.QPushButton('mass')
+        self.switchVolumMass.setToolTip('switch radius of spheres to represent relative masses or volumes')
```

Comment:
Wouldn't `Toggle` be better? In any case, the sentence should start with a capital letter.